superset-frontend/packages/superset-ui-chart-controls/src/components/Dropdown.tsx

```diff
@@ -17,5 +17,4 @@
  * under the License.
  */
 
-export { Dropdown } from 'antd';
-export type { DropDownProps } from 'antd/lib/dropdown';
+export { Dropdown, type DropDownProps } from 'antd-v5';
```

Comment:
is the plan to use what's in `src/components` once we move it to `@superset-ui/core` ?

Reply:
wondering what effect it may have on bundle sizes until we move things around ...